requirements-dev.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile
+# This file is autogenerated by pip-compile with python 3.8
```

Comment:
Hope this is compatible with minimum version we use and which is 3.7.6 atm.

Reply:
It's compatible. I've added Python 3.7 to Travis to make sure it stays compatible.